src/components/Sections/GoogleMapsSection/index.tsx

```diff
@@ -0,0 +1,199 @@
+import { Component, Prop } from "vue-property-decorator";
+import BaseComponent from "@/components/BaseComponent";
+import {
+  GoogleMapsSectionProps,
+  MapsPosition,
+  MapsLocation,
+} from "@/types/sections";
+import { LineSeparator } from "@/components";
+import { Loader } from "@googlemaps/js-api-loader";
+import "./style.css";
+// eslint-disable-next-line
+const markerIcon = require("../../../assets/Marker.png");
+
+@Component({
+  name: "GoogleMapsSection",
+})
+class GoogleMapsSection extends BaseComponent<GoogleMapsSectionProps> {
+  DEFAULT_STYLE: google.maps.MapTypeStyle[] = [
+    {
```

Comment:
I am not quite sure if this should be a property of the class since vue will add reactivity to it. I think it is better if you extract it into a constant outside of the class.